test/take.js

```diff
@@ -39,4 +39,19 @@ describe('take', function() {
     eq(R.into([], R.take(0), [1, 2, 3]), []);
   });
 
+  it('steps correct number of times', function() {
+    var takeXTimes = function(x) {
+      var counter = 0;
+      R.into(
+        [],
+        R.compose(R.map(function(x) { counter += 1; return x; }), R.take(x)),
```

Comment:
i agree that sounds like a good solution. let's add `sinon` to `devDependencies`